Makefile.prow

```diff
@@ -19,6 +19,10 @@ GOPATH := $(shell go env GOPATH)
 GOBIN := $(GOPATH)/bin
 GOSRC := $(GOPATH)/src
 
+# Prow settings for e2e tests
+OADP_E2E_DIR := /tmp/oadp-operator
+OADP_E2E_BRANCH := master
```

Comment:
speaking of override, why not oadp branch? too? `?=`
```suggestion
OADP_E2E_DIR ?= /tmp/oadp-operator
OADP_E2E_BRANCH ?= master
```

Reply:
if you are pulling velero branch=konveyor-dev you should use the master branch of oadp-operator. 
After master / default branches are complete we will consider backporting to oadp-1.4 and oadp-1.3.